Virtual Tourist/CoreDataStackManager.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  CoreDataStackManager.swift
+//  FavoriteActors
+//
+//  Created by Jason on 3/10/15.
+//  Copyright (c) 2015 Udacity. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+/**
+ * The CoreDataStackManager contains the code that was previously living in the 
+ * AppDelegate in Lesson 3. Apple puts the code in the AppDelegate in many of their
+ * Xcode templates. But they put it in a convenience class like this in sample code
+ * like the "Earthquakes" project.
+ *
+ */
+
+private let SQLITE_FILE_NAME = "VirtualTourist.sqlite"
+
+class CoreDataStackManager {
+    
+    
+    // MARK: - Shared Instance
+    
+    /**
+     *  This class variable provides an easy way to get access
+     *  to a shared instance of the CoreDataStackManager class.
+     */
+    class func sharedInstance() -> CoreDataStackManager {
+        struct Static {
```

Comment:
I thought there was a better way to do a sharedInstances in Swift? https://thatthinginswift.com/singletons/